libraries/chain/include/graphene/chain/protocol/asset_ops.hpp

```diff
@@ -442,6 +442,7 @@ namespace graphene { namespace chain {
       void            validate()const;
    };
 
+   /**
```

Comment:
This line is redundant.

Reply:
Shit, it seems I must be much much more careful when merging to resolve conflicts. On it ..